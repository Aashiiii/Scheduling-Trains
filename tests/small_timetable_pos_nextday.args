Clyde Windsor "12:12:11"
