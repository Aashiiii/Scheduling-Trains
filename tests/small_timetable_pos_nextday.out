The next train to Windsor from Clyde departs at 11:12:11
